range_pool_test.go

```diff
@@ -31,16 +35,14 @@ func Test_Service_Create_NumOne(t *testing.T) {
 
 	// Prepare the test variables.
 	ctx := context.TODO()
-	namespace := "test-namespace"
-	ID := "test-id"
```

Comment:
That will complain when you run the linter because now the magic string used more than 3 times. It will tell you you should create a constant for it or something.

Reply:
I don't know why but it didn't complain about repeating variables.